blueprints/app/files/_ts_eslint.config.mjs

```diff
@@ -145,5 +145,9 @@ export default ts.config(
         ...globals.node,
       },
     },
+    rules: {
+      'n/no-extraneous-import': 'warn',
+      'n/no-unsupported-features/node-builtins': 'warn',
+    },
```

Comment:
Fixes the errors seen in:

- https://github.com/ember-cli/ember-cli/actions/runs/15399484947/job/43328505950
- https://github.com/ember-cli/ember-cli/actions/runs/15399646183/job/43329122764
- https://github.com/ember-cli/ember-cli/actions/runs/15400185426/job/43330765063